client/taskingmanager.config.json

```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "EnvironmentConfig":{
-      "tmAPI": "http://tasking-manager-staging.eu-west-1.elasticbeanstalk.com/api/v1",
+      "tmAPI": "http://localhost:5000/api/v1",
```

Comment:
I think you may have left this committed while testing locally. (I have to edit these as well at times, but I think if we change the hard coded urls it may wreak havoc on the deployment process.)

Reply:
This is the test config, to our staging environment at thinkwhere which I will shortly be pulling down, so suggest we point this to your demo environment.

This won't impact deployment, as deployment uses the prod config

Reply:
Good to know, thanks!

Reply:
As mentioned previously, our staging environment has been removed, so I think localhost is most sensible default for now.